edx_django_utils/management/commands/manage_user.py

```diff
@@ -0,0 +1,183 @@
+"""
+Management command `manage_user` is used to idempotently create or remove
+Django users, set/unset permission bits, and associate groups by name.
+"""
+
+
+import random
+import string
+
+import django.dispatch
+from django.contrib.auth import get_user_model
+from django.contrib.auth.hashers import identify_hasher, is_password_usable
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.utils.translation import gettext as _
+
+manage_user_cmd = django.dispatch.Signal(providing_args=["user"])
```

Comment:
from Jira:
> The post save signal has a lot of things going on in it so I have created a custom signal which seems to be working fine.

Can you explain more? Unless there's a compelling reason not to, I think it's better to use the generic, built-in `User.post_save` signal instead defining a new one.

Reply:
Ia assume that you are talking about [this](https://github.com/edx/edx-platform/blob/29034dcbf5d7cad67419e9b7ee22c25d12b48ddc/common/djangoapps/student/models.py#L793) `post_save` signal. The `post_save` signal has a lot of conditions in there dealing with course enrollment which is why I thought that it would be safer to create a separate signal for this.

Reply:
I though it would be better to avoid issues with other Open edX users overriding or extending the `post_save` hook.

Reply:
@NIXKnight Thank you for considering that.

Keep in mind that the `User.post_save` signal is automatically fired when a User is saved anyway, so you would not need to fire it explicitly here. Also, signals are allowed to have multiple receivers, so handling the signal in edx-platform should not affect any other Open edX code that is listening for `User.post_save`. So, I don't foresee disruption if you were to handle `User.post_save` directly.